confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -168,9 +169,9 @@ def get_by_id(self, schema_id):
                 # cache it
                 self._cache_schema(result, schema_id)
                 return result
-            except:
+            except ClientError as e:
                 # bad schema - should not happen
-                raise ClientError("Received bad schema from registry.")
+                raise ClientError("Received bad schema (id %s) from registry: %s" % (schema_id, str(e)))
```

Comment:
Why cast to `e` to `str`? Why not rely on the implicit conversion?

Reply:
To favour __str__ over __repr__, but I guess __str__ kicks in here thanks to %s anyway